src/main/java/net/sf/jabref/gui/fieldeditors/EntryLinkListEditor.java

```diff
@@ -471,7 +471,7 @@ public Object getValueAt(int rowIndex, int columnIndex) {
                 case 1:
                     return entry.getLinkedEntry()
                             .map(bibEntry -> formatEntry(bibEntry, entry.getDataBase()))
-                            .orElse("Unknown entry");
+                            .orElse(Localization.lang("Unknown entry"));
```

Comment:
I'm not sure if you should translate strings that aren't localized. @JabRef/developers are these strings intentionally unlocalized?